homeassistant/components/todoist/calendar.py

```diff
@@ -221,10 +223,26 @@ async def handle_new_task(call: ServiceCall) -> None:
         if project_id is None:
             raise HomeAssistantError(f"Invalid project name '{project_name}'")
 
+        # Optional section within project
+        section_id: str | None = None
+        if SECTION_NAME in call.data:
+            section_name = call.data[SECTION_NAME].lower()
```

Comment:
Why do we call `lower` here, isn't lower case already guaranteed by the schema?
I think we should also, in a separate PR, remove the call to `lower` for the project name above.

Reply:
Sure, I'll update my PR to remove this. I'd gone with consistency, but you're right it's overkill.